Lib/xml/dom/minidom.py

```diff
@@ -80,6 +82,14 @@ def _get_lastChild(self):
             return self.childNodes[-1]
 
     def insertBefore(self, newChild, refChild):
+        """Insert a new DOM Node before an existing Node.
+
+        https://dom.spec.whatwg.org/#dom-node-insertbefore
+        The insertBefore(node, child) method, when invoked,
+        must return the result of pre-inserting node into
+        this before child.
```

Comment:
I think the docstring should document what the method does, not quote parts of a spec.

I would suggest simpler wording, but I don’t understand the existing one («the result of pre-inserting node into this before child» ?!)

Reply:
Addressed in [b97d812](https://github.com/python/cpython/pull/128477/commits/b97d812e6dde095b004d72465ed3f8837e46b96c)